theme_kea/views/snippets/s_color_blocks_2.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_color_blocks_2" inherit_id="website.s_color_blocks_2">
+        <!-- Box #1 -->
+        <xpath expr="//div[hasclass('col-lg-6')]" position="attributes">
+            <attribute name="class" add="oe_img_bg o_bg_img_center pt256 pb256" remove="" separator=" " />
+            <attribute name="style">background-image: url('/web/image/website.s_color_blocks_1_default_image'); background-position: 50% 0%;</attribute>
+        </xpath>
+        <!-- Box #1 - Icon -->
+        <xpath expr="//div[hasclass('col-lg-6')]//i" position="replace">
+        </xpath>  
+        <!-- Box #1 - Title -->
+        <xpath expr="//div[hasclass('col-lg-6')]//h2" position="replace">
+            <h2>Multiplayer</h2>
+        </xpath>
+        <!-- Box #1 - Text -->
+        <xpath expr="//div[hasclass('col-lg-6')]//p" position="replace">
+            <p>Conquer leaderboards, crush workouts, create with friends and more. Get lost in a growing library of VR classics and all-new experiences.</p>
+        </xpath>                
+        <!-- Box #1 - Button -->
+        <xpath expr="//div[hasclass('col-lg-6')]//a" position="attributes">
+            <attribute name="class" add="btn-secondary" remove="btn-primary" separator=" " />
+        </xpath>
+        <xpath expr="//div[hasclass('col-lg-6')]//a" position="replace">
```

Comment:
Same xpath as above.
If you want a text to be bold, wrap it with <b></b>, otherwise, it won't work with the editor.
Do you really need to add `.text-white`?

Same problems in Box #2